examples/experimental/armv7l.yaml

```diff
@@ -0,0 +1,20 @@
+# This example requires Lima v0.7.0 or later.
+
+arch: "armv7l"
+images:
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-amd64.img"
+  arch: "x86_64"
+- location: "https://cloud-images.ubuntu.com/releases/22.04/release/ubuntu-22.04-server-cloudimg-arm64.img"
+  arch: "aarch64"
```

Comment:
Doesn't make sense to add them to armv7l

Reply:
It was just showing the similarity, if it was to be merged. But can of course be removed, since arch was "hardcoded"